Sources/AssistiveButton/AssistiveButton.swift

```diff
@@ -96,19 +120,46 @@ final class AssistiveButton: UIButton {
         tapHandler()
     }
 
+    @objc private func hardwareShortcutPressed() {
+        tapHandler()
+    }
+
+    @objc private func panRecognizerHandler(_ recognizer: UIPanGestureRecognizer) {
```

Comment:
Not sure that `Handler` is appropriate word for method. What about `panGestureRecognized` or `panGestureDetected`?